src/backend/TrafficCourts/Workflow.Service/Services/OracleDataApiService.cs

```diff
@@ -23,9 +23,26 @@ public async Task<long> CreateDisputeAsync(Dispute dispute)
 
         return await _oracleDataApiClient.SaveDisputeAsync(dispute);
     }
+    public async Task<long> CreateFileHistoryAsync(FileHistory fileHistory)
+    {
+        try
+        {
+            return await _oracleDataApiClient.InsertFileHistoryAsync(fileHistory.TicketNumber, fileHistory);
+        } catch (Exception ex)

```

Comment:
ex variable is unused.
I think in C# you can just not include it
```suggestion
        } catch (Exception)
```